src/serialization/serialization.rs

```diff
@@ -176,13 +183,19 @@ mod benchmarks;
 mod v2_serializer;
 pub use self::v2_serializer::{V2Serializer, V2SerializeError};
 
+#[path = "v2_deflate_serializer.rs"]
```

Comment:
You'd think that, and yet without this...

```
error: cannot declare a new module at this location
   --> src/serialization/serialization.rs:186:1
    |
186 | mod v2_deflate_serializer;
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
    = note: for more information, see issue #37872 <https://github.com/rust-lang/rust/issues/37872>
```